homeassistant/components/stream/core.py

```diff
@@ -35,6 +36,19 @@
 PROVIDERS: Registry[str, type[StreamOutput]] = Registry()
 
 
+class Orientation(IntEnum):
+    """Orientations for stream transforms."""
+
+    ONE = 1
+    TWO = 2
+    THREE = 3
+    FOUR = 4
+    FIVE = 5
+    SIX = 6
+    SEVEN = 7
+    EIGHT = 8
```

Comment:
I like the use of an enum, but could we match the names with the orientation "type"?
Or at the very least add a doctype to each element.

```python
    NO_TRANSFORM = 1
    MIRROR = 2
    ROTATE_180 = 3
    FLIP = 4
    ROTATE_LEFT_AND_FLIP = 5
    ROTATE_LEFT = 6
    ROTATE_RIGHT_AND_FLIP = 7
    ROTATE_RIGHT = 8
```
or
```python
    ONE = 1
    """No transform."""
    TWO = 2
    """Mirror."""
```

Reply:
That's a good point - I thought it would be hard to name them and forgot I already had named them previously 😄